sorting/threeway_merge_sort.cpp

```diff
@@ -0,0 +1,114 @@
+/**
+ *  \addtogroup sorting Sorting Algorithms
```

Comment:
```suggestion
```
This add to group tag was never mentioned in the contributing file where are you getting it from?